packages/web-components/src/components/ic-select/ic-select.e2e.ts

```diff
@@ -1471,6 +1481,15 @@ describe("ic-select", () => {
     expect(value).toBe(undefined);
   });
 
+  it("shouldnt display no results found on initial load and setting default value on disable filter", async () => {
```

Comment:
```suggestion
  it("shouldn't display no results found on initial load if setting default value and disable filter", async () => {
```

Reply:
Perhaps it should be 'should not' for consistency as most e2e tests are using 'should not'.